diff -urN awesfx-0.4.3c.orig/Imakefile awesfx-0.4.3c/Imakefile
--- awesfx-0.4.3c.orig/Imakefile	Mon Jan 17 00:07:48 2000
+++ awesfx-0.4.3c/Imakefile	Wed Feb  2 15:44:23 2000
@@ -4,6 +4,7 @@
 
 /*CDEBUGFLAGS=-g*/
 CDEBUGFLAGS=-Wall
+CCOPTIONS=$(RPM_OPT_FLAGS)
 
 #include "awelib/config.h"
 
@@ -11,10 +12,10 @@
 SUBDIRS = awelib include
 
 /* binary and manual directories */
-INSTDIR = /usr/local
+INSTDIR = /usr
 
 BINDIR = $(INSTDIR)/bin
-MANDIR = $(INSTDIR)/man/man1
+MANDIR = $(_MANDIR)/man1
 MANSUFFIX = 1
 
 #define PassCDebugFlags CDEBUGFLAGS=$(CDEBUGFLAGS) INSTDIR=$(INSTDIR)
@@ -37,31 +38,31 @@
 
 NormalProgramTarget(sfxload, sfxload.o seq.o, $(DEPLIB),  $(AWELIB), -lm)
 AllTarget(sfxload)
-InstallProgram(sfxload, $(BINDIR))
-InstallManPage(sfxload, $(MANDIR))
+InstallProgram(sfxload,$(BINDIR))
+InstallManPage(sfxload,$(MANDIR))
 
 NormalProgramTarget(setfx, setfx.o seq.o, $(DEPLIB),  $(AWELIB), -lm)
 AllTarget(setfx)
-InstallProgram(setfx, $(BINDIR))
+InstallProgram(setfx,$(BINDIR))
 
 NormalProgramTarget(sf2text, sf2text.o, $(DEPLIB), $(AWELIB), -lm)
 AllTarget(sf2text)
-InstallProgram(sf2text, $(BINDIR))
+InstallProgram(sf2text,$(BINDIR))
 
 NormalProgramTarget(text2sf, text2sf.o, $(DEPLIB),  $(AWELIB), -lm)
 AllTarget(text2sf)
-InstallProgram(text2sf, $(BINDIR))
+InstallProgram(text2sf,$(BINDIR))
 
 NormalProgramTarget(gusload, gusload.o seq.o,,, -lm)
 AllTarget(gusload)
-InstallProgram(gusload, $(BINDIR))
+InstallProgram(gusload,$(BINDIR))
 
 NormalProgramTarget(sfxtest, sfxtest.o seq.o,,, -lm)
 AllTarget(sfxtest)
-InstallProgram(sfxtest, $(BINDIR))
+InstallProgram(sfxtest,$(BINDIR))
 
 NormalProgramTarget(aweset, aweset.o seq.o, $(DEPLIB), $(AWELIB), -lm)
 AllTarget(aweset)
-InstallProgram(aweset, $(BINDIR))
+InstallProgram(aweset,$(BINDIR))
 
 NamedMakeSubdirs($(DEPLIB),$(SUBDIRS))
diff -urN awesfx-0.4.3c.orig/awelib/Imakefile awesfx-0.4.3c/awelib/Imakefile
--- awesfx-0.4.3c.orig/awelib/Imakefile	Mon Jan 17 00:07:48 2000
+++ awesfx-0.4.3c/awelib/Imakefile	Wed Feb  2 15:46:43 2000
@@ -6,7 +6,9 @@
 
 LIBVER = 0.4.3
 
-INSTDIR = /usr/local
+CCOPTIONS=$(RPM_OPT_FLAGS)
+
+INSTDIR = /usr
 
 LIBDIR = $(INSTDIR)/lib
 INCLUDES += -I../include
@@ -24,18 +26,18 @@
 
 %.o: %.c
 	NormalSharedLibObjCompile()
-SharedLibraryTarget(awe,$(LIBVER), $(SHOBJS), ., .)
+SharedLibraryTarget(awe,$(LIBVER),$(SHOBJS), ., .)
 #ifdef INSTALL_AWELIB
-InstallSharedLibrary(awe,$(LIBVER), $(LIBDIR))
+InstallSharedLibrary(awe,$(LIBVER),$(LIBDIR))
 #endif
 
 #else
 
 %.o: %.c
 	NormalLibObjCompile()
-NormalLibraryTarget(awe, $(SHOBJS))
+NormalLibraryTarget(awe,$(SHOBJS))
 #ifdef INSTALL_AWELIB
-InstallLibrary(awe, $(LIBDIR))
+InstallLibrary(awe,$(LIBDIR))
 #endif
 
 #endif
diff -urN awesfx-0.4.3c.orig/include/Imakefile awesfx-0.4.3c/include/Imakefile
--- awesfx-0.4.3c.orig/include/Imakefile	Mon Jan 17 00:07:48 2000
+++ awesfx-0.4.3c/include/Imakefile	Mon Jan 31 14:56:41 2000
@@ -1,11 +1,11 @@
 #include "../awelib/config.h"
 
-INSTDIR = /usr/local
+INSTDIR = /usr
 INCDIR = $(INSTDIR)/include/awe
 
 HEADERS = awebank.h aweseq.h awe_parm.h sffile.h itypes.h sfitem.h sflayer.h\
 	sfopts.h slist.h util.h
 
 #ifdef INSTALL_AWELIB
-InstallMultiple($(HEADERS), $(INCDIR))
+InstallMultiple($(HEADERS),$(INCDIR))
 #endif
